### PR TITLE
Metadata fixes: updated time, owner, group

### PR DESCRIPTION
**Pull request by crufter**

### CODE REVIEW

File: service/runtime/server/handler.go [hunk 1]
```diff
@@ -136,6 +142,23 @@ func (r *Runtime) Update(ctx context.Context, req *pb.UpdateRequest, rsp *pb.Upd
...
+		}
+		service.Metadata["owner"] = account.ID
+		// This is a hack - we don't want vanilla `micro server` users where the auth is noop
+		// to have long uuid as owners, so we put micro here - not great, not terrible.
+		if auth.DefaultAuth.String() == "noop" {
+			service.Metadata["owner"] = "micro"
+		}
+		service.Metadata["group"] = account.Issuer
+	}
+	service.Metadata["started"] = time.Now().Format(time.RFC3339)
```
Comment (ben-toogood) [inline service/runtime/server/handler.go:159]: service.Metadata could be nil

File: service/runtime/server/handler.go [hunk 2]
```diff
@@ -136,6 +142,23 @@ func (r *Runtime) Update(ctx context.Context, req *pb.UpdateRequest, rsp *pb.Upd
 	return nil
 }
 
+func setupServiceMeta(ctx context.Context, service *runtime.Service) {
+	account, accOk := goauth.AccountFromContext(ctx)
+	if accOk && account != nil {
```
Comment (ben-toogood) [inline service/runtime/server/handler.go:147]: Account should never be nil if accOK is true

File: service/runtime/server/handler.go (hunk 1 above)
Comment (crufter) [reply service/runtime/server/handler.go:159]: There is a check for that above

Comment (crufter) [reply service/runtime/server/handler.go:159]: ... at the wrong place

File: service/runtime/server/handler.go (hunk 2 above)
Comment (crufter) [reply service/runtime/server/handler.go]: Gotcha

**Pull request merged**